### ISSUE TITLE
T4 - Mi modelo no se entrena ?

### ISSUE THREAD

**Issue by EleaTriesToCode** [IIC2613-Inteligencia-Artificial-2022-2/Syllabus#91]
Hola ! 

Estoy haciendo la parte de las redes neuronales densas. Eso es el modelo que tengo ahora :

# Construct model 
model = Sequential()

model.add(Dense(256, input_shape=(40,)))
model.add(Activation('relu'))
model.add(Dropout(0.5))

model.add(Dense(256))
model.add(Activation('relu'))
model.add(Dropout(0.5))

model.add(Dense(10))
model.add(Activation('softmax'))

# Por último mostramos un resumen del modelo
model.summary()

Con un batch size de 32 y unas 100 epocas.

He transformado el y para que sea un array de arrays [0,0,...,1,0...] por ejemplo (donde 1 esta en el indice de la clase). He intentado normalizar los valores de X...

Mi problema es que tengo un accuracy incredible mala (como 1% no mas) y que no aumenta durante el training...
No sé lo que puedo estar haciendo mal ? 

Si alguien ha encontrado este problema o tiene una idea de lo que puedo haber olvidado agredeciera mucho la ayuda ! He leido muchas veces la ayudantia y no sé que me falta :0

**Comment by tcouso** [comment]
Hola! Disculpa la demora en responder. Qué optimizador estás usando? Buscando en stack overflow encontré alguien que tenía un problema parecido y tenía que ver con el optimizador usado
